nodejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "http_ece",
-  "version": "0.6.3",
+  "version": "0.6.5",
```

Comment:
The padding change renders this version not backwards compatible. Since it's still not officially release, I'd STRONGLY urge at least a minor step and make this `0.7`.